### DISCUSSION TITLE
Minidexed Modular

### DISCUSSION THREAD

**Discussion by zsoltmar** [probonopd/MiniDexed#383] [Ideas]
Been thinking about creating a module using Minidexed. How might one go about using CV (Pitch + Gate) to trigger the synth? Would there be a way to do it without having to do a CV to MIDI conversion?

**Comment by diyelectromusic** [comment]
The DX7 is a digital MIDI synth and doesn't understand CV/Gate as I understand things, so no, I think you'll have to use a CV to MIDI conversion somewhere in the chain...

See more on this here: https://github.com/probonopd/MiniDexed/issues/28

Kevin
